### PR TITLE
Calibration check and fixes

### PR DESCRIPTION
**Pull request by profxj**
This continues on from PR#160 and fixes a few issues
related to that, and updates additional documentation.

It also makes modifications to insure calcheck runs
successfully.

### CODE REVIEW

File: pypit/arsort.py [hunk 1]
```diff
@@ -978,7 +985,7 @@ def write_setup(setup_dict, use_json=False):
     # Write
     setup_file, nexist = get_setup_file()
     if nexist == 1:
-        msgs.error("Cannot over-write existing setup file.  If you wish to, remove and rerun")
```
Comment (rcooke-ast) [inline pypit/arsort.py]: Would it be worthwhile checking if "output overwrite" is set to True before overwriting?

**Comment by rcooke-ast** [comment]
I made two changes directly to the PR, and made one comment. Feel free to merge when ready.

File: pypit/arsort.py (hunk 1 above)
Comment (profxj) [inline pypit/arsort.py]: At this stage, I think it is fine to over-write.

If we want to be strict about `output overwrite True`, that should
be a separate PR and touch many aspects.

Comment (rcooke-ast) [inline pypit/arsort.py]: OK, agreed!

**Comment by profxj** [comment]
Merging
